packages/3proxy-win32/PKGBUILD

```diff
@@ -20,4 +20,5 @@ package() {
   rm 3proxy-${pkgver}.zip
 
   cp -a * "$pkgdir/opt/windows/archive/3proxy-win32"
+
```

Comment:
The line isn't here, it's in the end of the script.

Reply:
Sorry, fatigue affects :)